src/app_modes/topic_managment.rs

```diff
@@ -0,0 +1,365 @@
+                input::LEFT => self.shift_active_element_left(),
+                input::ROTATE_LEFT => self.selection_mode = true,
+                input::CONFIRM => self.save(),
+                _ => (),
+            }
+        }
+    }
+
+    fn get_keymap(&self) -> Vec<[String; 2]> {
+        vec![
+            [
+                input::UP.to_string(),
+                "Select the previous item in the active list".to_string(),
+            ],
+            [
+                input::DOWN.to_string(),
+                "Select the next item in the active list".to_string(),
+            ],
+            [
+                input::RIGHT.to_string(),
+                "Shift an element to the right if the supported topic list is active".to_string(),
+            ],
+            [
+                input::LEFT.to_string(),
+                "Shift an element to the left if the active list is active".to_string(),
+            ],
+            [
+                input::ROTATE_RIGHT.to_string(),
+                "Change the list where items are selected to the active topics list".to_string(),
+            ],
+            [
+                input::ROTATE_LEFT.to_string(),
+                "Change the list where items are selected to the supported topics list".to_string(),
+            ],
+            [
+                input::CONFIRM.to_string(),
+                "Save to config".to_string(),
+            ],
+        ]
+    }
+
+    fn get_name(&self) -> String {
+        "Topic Manager".to_string()
+    }
+}
+
+impl<B: Backend> Drawable<B> for TopicManager {
+    fn draw(&self, f: &mut Frame<B>) {
+        let title_text = vec![Spans::from(Span::styled(
+            "Topic Manager",
+            Style::default().fg(Color::Red).add_modifier(Modifier::BOLD),
+        ))];
+        let areas = Layout::default()
+            .direction(Direction::Vertical)
+            .horizontal_margin(20)
+            .constraints(
+                [
+                    Constraint::Length(3), // Title + 2 borders
+                    Constraint::Length(2),
+                    Constraint::Min(1), // Table + header + space
+                ]
+                .as_ref(),
+            )
+            .split(f.size());
+
+        let left_chunks = Layout::default()
+            .direction(Direction::Horizontal)
+            .margin(1)
+            .constraints(
+                [
+                    Constraint::Percentage(50),
+                    Constraint::Percentage(50),
+                ]
+                .as_ref(),
+            )
+            .split(areas[2]);
+        // Widget creation
+        let title = Paragraph::new(title_text)
+            .block(Block::default().borders(Borders::ALL))
+            .style(Style::default().fg(Color::White))
+            .alignment(Alignment::Center)
+            .wrap(Wrap { trim: false });
+        let items: Vec<ListItem>= self.availible_topics.items.iter().map(|i| ListItem::new(
+                format!("{} / {}", i[0], i[1])
```

Comment:
don't use a slash between the topic and it's type, since slashes are normally used inside a topic name. How about an arrow `->`? or a dash? or something like that

Reply:
i will use a :